packages/airplane/internal/params/params_serialized.ts

```diff
@@ -73,15 +73,29 @@ export const serializeParamValues = (paramValues: InputParamValues): SerializedP
   return Object.fromEntries(newParamEntries);
 };
 
-export const serializeParamValue = (paramValue: InputParamValue): SerializedParamValue => {
+export const serializeParamValue = (
+  paramValue: InputParamValue | InputParamValue[],
+  multi?: boolean
+): SerializedParamValue | SerializedParamValue[] => {
+  if (multi) {
+    if (paramValue === undefined || paramValue === null) {
+      return paramValue;
+    }
+    return ((paramValue as InputParamValue[]) ?? []).map((v) => {
```

Comment:
You shouldn't need this since you are already checking if it's `null`

```suggestion
    return (paramValue as InputParamValue[]).map((v) => {
```